### DISCUSSION TITLE
link para a clase de hoy......

### DISCUSSION THREAD

**Discussion by Yvillacis** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#481] [Q&A]
Alguien tiene acceso a la semana 6 del curso. No me aparece la semana 6 y no se donde esta el link para la clase de hoy.
gracias de antemano.

**Comment by Ingrzs** [comment]
no ahi link ni nada tal ves el mc tuvo un contratiempo, habra que esperar...

**Reply by Yvillacis** [reply]
Muchas gracias por la informacion, pense que era solo yo la del problema....

**Comment by doguedogue** [comment]
https://teams.microsoft.com/l/meetup-join/19%3ameeting_Y2M0ZGM3MDItYTNiYi00ZTZiLWIyMWUtZTY2MjkwNTNmYTFh%40thread.v2/0?context=%7b%22Tid%22%3a%224ae54b05-b77e-4224-aef1-8661422e0816%22%2c%22Oid%22%3a%2296eeb66f-b984-4083-8ea6-74c34a07a34a%22%2c%22IsBroadcastMeeting%22%3atrue%7d
